webclient/src/forms/ResetPasswordForm/ResetPasswordForm.tsx

```diff
@@ -9,18 +9,31 @@ import { InputField, KnownHosts } from 'components';
 import { FormKey } from 'types';
 
 import './ResetPasswordForm.css';
+import { useReduxEffect } from '../../hooks';
+import { ServerTypes } from '../../store';
 
 const ResetPasswordForm = (props) => {
   const { dispatch, handleSubmit } = props;
 
+  const [errorMessage, setErrorMessage] = useState(false);
+
+
   const onHostChange = ({ host, port }) => {
     dispatch(change(FormKey.RESET_PASSWORD, 'host', host));
     dispatch(change(FormKey.RESET_PASSWORD, 'port', port));
   }
 
+  useReduxEffect(() => {
+    setErrorMessage(true);
+  }, ServerTypes.RESET_PASSWORD_FAILED, []);
```

Comment:
We should inspect the error response object for a fail reason enum or string. Same with the other form failure effect. We can pass those into the action from the websocket layer and pick them up here in the component redux effect.

Reply:
the error response object doesn't contain such information right now, the response is either respOk or respFunctionNotAllowed without details